### PR TITLE
OSDOCS-11764 OCP Release Notes 4.15.29

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue: [OSDOCS-11764](https://issues.redhat.com/browse/OSDOCS-11764)

Link to [docs preview](https://80914--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-29_release-notes)

QE review:
Not needed

Additional information: Advisory links will not work until August 28.

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2740,7 +2740,32 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[source,terminal]
+----
+$ oc adm release info 4.15.29 --pullspecs
+----
+
+[id="ocp-4-15-29-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the ordering of Network Interface Controllers (NICs) in the cloud provider was non-deterministic and could result in the node choosing the wrong NIC for communication with the cluster. With this update, the ordering will always be consistent, which removes the possibility for random ordering making the node networking fail. (link:https://issues.redhat.com/browse/OCPBUGS-38577[*OCPBUGS-38577*])
```
Comment (mburke5678) [inline release_notes/ocp-4-15-release-notes.adoc:2762]: Maybe tighten this up a bit, the second sentence is a bit awkward to me. 
```suggestion
* Previously, the ordering of Network Interface Controllers (NICs) in the cloud provider was non-deterministic, which could result in the node using the wrong NIC for communication with the cluster. With this update, the ordering is always be consistent. This fix prevents the random ordering that was causing the node networking fail. (link:https://issues.redhat.com/browse/OCPBUGS-38577[*OCPBUGS-38577*])
```

**Comment by mburke5678** [comment]
@cbippley One suggestion. Otherwise LGTM

**Review by jldohmann — APPROVED** [review]
/lgtm

**Pull request merged**